kustomize/base/shard/resources/managedprocessors.com.redhat.service.smartevents-v1.yml

```diff
@@ -0,0 +1,62 @@
+# Generated by Fabric8 CRDGenerator, manual edits might get overwritten!
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: managedprocessors.com.redhat.service.smartevents
+spec:
+  group: com.redhat.service.smartevents
+  names:
+    kind: ManagedProcessor
+    plural: managedprocessors
+    shortNames:
+    - mproc
+    - mp
+    singular: managedprocessor
+  scope: Namespaced
+  versions:
+  - name: v2alpha1
+    schema:
+      openAPIV3Schema:
+        properties:
+          spec:
+            properties:
+              id:
+                type: string
+              bridgeId:
+                type: string
+              name:
+                type: string
+              shardId:
```

Comment:
@lucamolteni We don't need `shardId` in the spec unless there was a good reason you'd identified?